plugin/trino-databend/src/test/java/io/trino/plugin/databend/TestDatabendConnectorTest.java

```diff
@@ -0,0 +1,1256 @@
+import static java.util.stream.IntStream.range;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assumptions.abort;
+
+public class TestDatabendConnectorTest
+        extends BaseJdbcConnectorTest
+{
+    protected TestingDatabendServer databendServer;
+
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        databendServer = closeAfterClass(new TestingDatabendServer(TestingDatabendServer.DATABEND_DEFAULT_IMAGE));
+        return DatabendQueryRunner.builder(databendServer)
+                .addConnectorProperty("databend.connection-timeout", Duration.valueOf("60s").toString())
+                .addConnectorProperty("databend.presigned-url-disabled", "true")
+                .setInitialTables(REQUIRED_TPCH_TABLES)
+                .build();
+    }
+
+    @Override
+    protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
+    {
+        return switch (connectorBehavior) {
+            // Tests requires s3 presigned url to be disabled
+            /*
+            the query plan of databend is like: Output[columnNames = [_col0, _col1]]
+│   Layout: [pfgnrtd:double, pfgnrtd_1:real]
+│   _col0 := pfgnrtd
+│   _col1 := pfgnrtd_1
+└─ TableScan[table = databend:Query[SELECT corr("t_double", "u_double") AS "_pfgnrtd_0", corr("v_real", "w_real") AS "_pfgnrtd_1" FROM "tpch"."test_corr_pushdownjp0d6s0d4r"] columns=[_pfgnrtd_0:double:Float64, _pfgnrtd_1:real:Float32]]
+       Layout: [pfgnrtd:double, pfgnrtd_1:real]
+       pfgnrtd := _pfgnrtd_0:double:Float64
+       pfgnrtd_1 := _pfgnrtd_1:real:Float32
+
+            **/
+            case SUPPORTS_AGGREGATION_PUSHDOWN,
+                    SUPPORTS_JOIN_PUSHDOWN,
+                    SUPPORTS_LIMIT_PUSHDOWN,
+                    SUPPORTS_AGGREGATION_PUSHDOWN_CORRELATION,
+                    SUPPORTS_AGGREGATION_PUSHDOWN_REGRESSION,
+                    SUPPORTS_AGGREGATION_PUSHDOWN_STDDEV,
+                    SUPPORTS_DYNAMIC_FILTER_PUSHDOWN,
+                    SUPPORTS_TOPN_PUSHDOWN -> false;
+
+            case SUPPORTS_ARRAY,
+//                    SUPPORTS_DELETE,
+                    SUPPORTS_COMMENT_ON_COLUMN,
+                    SUPPORTS_NEGATIVE_DATE, // min date is 0001-01-01
+//                    SUPPORTS_RENAME_TABLE,
+                    SUPPORTS_ROW_TYPE,
+                    SUPPORTS_DROP_NOT_NULL_CONSTRAINT,
+                    SUPPORTS_ROW_LEVEL_DELETE,
+                    SUPPORTS_RENAME_TABLE_ACROSS_SCHEMAS,
+                    SUPPORTS_INSERT,
+                    SUPPORTS_NATIVE_QUERY,
+                    SUPPORTS_SET_COLUMN_TYPE,
+                    SUPPORTS_UPDATE -> false;
+
+            default -> super.hasBehavior(connectorBehavior);
+        };
+    }
+
+    @Test
+    @Disabled
+    @Override
+    public void testAddAndDropColumnName() {}
```

Comment:
Why do we disable many tests? Please leave comments in each test. Also, we should verify the behavior instead of skipping tests.